### PR TITLE
fix: openapi-validation for corda plugin

### PR DESCRIPTION
**Pull request by elenaizaguirre**
add missing validation for corda plugin rest endpoints

Closes: #1293

Closes: #847 

Signed-off-by: Elena Izaguirre <e.izaguirre.equiza@accenture.com>

### CODE REVIEW

File: packages/cactus-plugin-ledger-connector-corda/src/main-server/kotlin/gen/kotlin-spring/src/main/kotlin/org/hyperledger/cactus/plugin/ledger/connector/corda/server/model/NetworkMapV1Request.kt [hunk 1]
```diff
@@ -1,39 +0,0 @@
-package org.hyperledger.cactus.plugin.ledger.connector.corda.server.model
```
Comment (petermetz) [inline packages/cactus-plugin-ledger-connector-corda/src/main-server/kotlin/gen/kotlin-spring/src/main/kotlin/org/hyperledger/cactus/plugin/ledger/connector/corda/server/model/NetworkMapV1Request.kt:1]: @elenaizaguirre Question: Why were the contents of this file removed? I thought the code generator would fill this based on the openapi.json.

**Review by takeutak — APPROVED** [review]

File: packages/cactus-plugin-ledger-connector-corda/src/main-server/kotlin/gen/kotlin-spring/src/main/kotlin/org/hyperledger/cactus/plugin/ledger/connector/corda/server/model/NetworkMapV1Request.kt (hunk 1 above)
Comment (elenaizaguirre) [reply packages/cactus-plugin-ledger-connector-corda/src/main-server/kotlin/gen/kotlin-spring/src/main/kotlin/org/hyperledger/cactus/plugin/ledger/connector/corda/server/model/NetworkMapV1Request.kt:1]: @petermetz The object definition in `openapi.json` is empty, it simply defines it as object type. I don't know when this file was generated but if you delete the `model` folder and generate it with the `generate-server` script in `package.json` (it should probably also take care of deleting that folder) you will see that `NetworkMapV1Request` and `NetworkMapV1Response` are not generated, but the new connector is working in all tests.

**Comment by elenaizaguirre** [comment]
@petermetz ci execution is failing in the modified tests in the [last commit](https://github.com/hyperledger/cactus/commit/4beab22082e26e5e033711ea5ed6b46f3f5eb52a). Is there anything I can do to fix it and pass checks?

File: packages/cactus-plugin-ledger-connector-corda/src/main-server/kotlin/gen/kotlin-spring/src/main/kotlin/org/hyperledger/cactus/plugin/ledger/connector/corda/server/model/NetworkMapV1Request.kt (hunk 1 above)
Comment (petermetz) [reply packages/cactus-plugin-ledger-connector-corda/src/main-server/kotlin/gen/kotlin-spring/src/main/kotlin/org/hyperledger/cactus/plugin/ledger/connector/corda/server/model/NetworkMapV1Request.kt:1]: @elenaizaguirre My bad, this is an issue with the build itself then, sorry. I created this for us to follow up on the issue later on more: https://github.com/hyperledger/cactus/issues/1559

**Review by petermetz — APPROVED** [review]
@elenaizaguirre LGTM, thank you!

**Pull request merged**